src/routes/(modeles-services)/services/[slug]/orienter/demande/orientation-form.svelte

```diff
@@ -170,7 +171,7 @@
                 {#each $orientation.situation as label}
                   {#if label === OTHER_LABEL}
                     <li class="text-gray-text">
-                      {label} : {$orientation.situationOther}
+                      {label}&nbsp;:&nbsp;{$orientation.situationOther}
```

Comment:
```suggestion
                      {label}&nbsp;: {$orientation.situationOther}
```
Hum, non, pas après les deux points...